### ISSUE TITLE
compile errors

### ISSUE THREAD

**Issue by matejthetree** [mbarrben/android_dialer_replacement#1]
I am trying to compile your app with fresh android studio installation.

First I need to update kotlin to 1.3.21 and then I need to change the link from jre to jdk.

After I run that it still doesnt work.

Can you help the noob :D

**Comment by felipemessersmith** [comment]
Of course! Don't worry, I'm here to help. Let's tackle this step by step:

To update Kotlin to version 1.3.21, go to your project's build.gradle file and update the Kotlin version in the dependencies section to 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21'.

Now, you need to change the link from JRE to JDK. In Android Studio, go to File > Project Structure. Under SDK Location, make sure the JDK location is set correctly.

After making these changes, try running the app again. If it still doesn't work, make sure to check the error messages in the logs or console [betflix](https://betflix.me/). Feel free to share the error details with me, and I'll do my best to assist you further.

**Comment by bcsk** [comment]
Of course, I'd be happy to help you with your Android Studio installation. Here are the steps you can follow to update Kotlin to version 1.3.21 and change the Java Development Kit (JDK) in Android Studio:

Updating Kotlin to 1.3.21:

Open Android Studio.

Go to "File" > "Project Structure."

In the "Project Structure" dialog, on the left sidebar, click on "Project."

In the "Project" pane, you should see an option for "Kotlin Version." Click on it.

In the dropdown menu for "Kotlin Version," select "1.3.21" or the version you want to use.

Click "Apply" and then "OK" to save the changes.

Changing JDK in Android Studio:

Go to "File" > "Project Structure."

In the "Project Structure" dialog, on the left sidebar, click on "Project."

Under "Project SDK," click on the dropdown menu and select the desired JDK version. It's recommended to use a JDK 8 version, as it's well-supported for Android development.

Click "Apply" and then "OK" to save the changes.

Close and reopen Android Studio to ensure the changes take effect.

Now, after making these [mickeyminors](https://mickeyminors.pk/) changes, if you're still experiencing issues, please provide more details about the specific problems you're encountering, any error messages you're seeing, and the steps you've taken so far. This additional information will help me assist you further.

**Comment by Abaudiovisualpk** [comment]
I wanted to start a discussion about AB Audiovisual.pk in Pakistan, a platform that's been making waves in the audiovisual industry. It's fascinating how this platform has become a hub for creativity, innovation, and collaboration in our country. https://absecuritysol.com/

**Comment by Kidzaad12** [comment]
Navigating through your website was a breeze, and I love the intuitive layout. It's clear that you've put thought into creating a user-friendly experience. The detailed product descriptions and sizing guides also make it easy for parents like me to make informed choices for our little fashionistas. [Kidzaad.pk ](https://kidzaad.pk/)
